code/datums/votes/map_vote.dm

```diff
@@ -7,9 +7,9 @@
 
 /datum/vote/map_vote/New()
 	. = ..()
```

Comment:
this doesn't do anything anymore so delete it

Reply:
This breaks map vote entirely. default_choices needs to have something in it or it won't be accessible.

```
/datum/vote/proc/is_accessible_vote()
	return !!length(default_choices)
```

![image](https://github.com/user-attachments/assets/92c29179-005c-46c4-9fad-d3114eb5df22)